### PR TITLE
Gemfile additions, seeder and dinosaur table  migration, dino model

### PR DESCRIPTION
**Pull request by mcasey5216**
See title.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,11 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+# t.string :name, null: false
+# t.string :location_found, null: false
+# t.string :info_url, null: false
+# t.string :photo_url
+# t.belongs_to :user
+
+Dinosaur.create!(name: "Giganotosaurus", location_found: "South America", info_url: "https://en.wikipedia.org/wiki/Giganotosaurus")
```
Comment (davidrf) [inline db/seeds.rb]: Possible fix

``` ruby
Dinosaur.create!(
  name: "Giganotosaurus",
  location_found: "South America",
  info_url: "https://en.wikipedia.org/wiki/Giganotosaurus"
)
```

File: log/development.log [hunk 2]
```diff
@@ -0,0 +1,101 @@
+  [1m[36m (454.7ms)[0m  [1mCREATE TABLE "schema_migrations" ("version" character varying NOT NULL) [0m
```
Comment (davidrf) [inline log/development.log]: Add `log/` to your `.gitignore` file to prevent these from being added.

**Comment by mcasey5216** [comment]
how about now?

File: log/development.log (hunk 2 above)
Comment (davidrf) [inline log/development.log]: Sorry I think it's supposed to be `/log` or play around with it until this file isn't uploaded anymore

**Pull request merged**